src/libraries/System.Text.RegularExpressions/tests/FunctionalTests/MonoRegexTests.cs

```diff
@@ -283,6 +283,8 @@ static IEnumerable<(string Pattern, RegexOptions Options, string Input, string R
                 yield return (@"a[^]b]c", RegexOptions.IgnoreCase, "ADC", "Pass. Group[0]=(0,3)");
                 yield return (@"ab|cd", RegexOptions.IgnoreCase, "ABC", "Pass. Group[0]=(0,2)");
                 yield return (@"ab|cd", RegexOptions.IgnoreCase, "ABCD", "Pass. Group[0]=(0,2)");
+                yield return (@"a{2}|a{3}", RegexOptions.None, "aaa", "Pass. Group[0]=(0,2)");
+                yield return (@"a{3}|a{2}", RegexOptions.None, "aaa", "Pass. Group[0]=(0,3)");
```

Comment:
What inspired these two additions?

Reply:
I can't remember! Some past bug, apparently fixed now, with quantifiers in an alternation. But they do seem to fit with the others.

Reply:
My concern is that this file is primarily populated with tests inherited from the original mono repo, and in turn it inherited them from perl.  It'd be strange, for example, to add new tests to our ports of those from rust or pcre.  Plus, if the issue was fixed, there's a really good chance we have tests for the issue elsewhere that were added when that issue was fixed.  I don't object to new tests being added, of course, but it'd be good to put them in a better place and ensure we don't already have coverage for the issue.

Reply:
moved to a more reasonable file, they seem complementary there